composer.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "ezsystems/privacy-cookie-bundle",
+    "description": "Privacy cookie banner integration bundle into eZ Publish/eZ Platform",
+    "keywords": ["cookies", "privacy", "policy"],
+    "license": "GPL-2.0",
+    "authors": [
+        {
+          "name": "eZ Publish dev-team & eZ Community",
+          "homepage": "https://github.com/ezsystems/ezpublish-community/contributors"
+        }
+    ],
+    "require": {
+        "symfony/symfony": "~2.5",
```

Comment:
I'm not sure about this. I think this bundle would work with Symfony 2.3 also. Older releases are not supported, so we can forget about them.